.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
```

Comment:
I chose to add a few CI tests to verify the script was passing the assignment conditions using CircleCI.

If you choose you can:

1. Create a CircleCI project to run these tests.
2. View the tests run on my own forked project [at the acohen-code-test branch.](https://github.com/cohenaj194/platform-code-test/tree/)
3. Install the cli tool for CircleCI and then run:
```
circleci local execute --job rubocop
circleci local execute --job rspec
```